slidez-core/src/main/resources/application.properties

```diff
@@ -25,3 +24,9 @@ spring.flyway.enabled=true
 #-------------------------
 springdoc.api-docs.path=/api-docs
 springdoc.swagger-ui.path=/swagger-ui
+#-------------------------
+# Sentry
+#-------------------------
+sentry.dsn=https://b579813c296046399a14368064320494@o938867.ingest.sentry.io/5888744
+sentry.in-app-includes=com.binarystudio.academy.slidez
+
```

Comment:
Move it to staging properties, as we don't want to get exception message every time somebody debug something :)

Reply:
Yeah, it was there, but after refactoring it didn't work this way. So i found the possible variant with sentry.properties file and it worked, but in call with Kirill he told me to put sentry dsn key only for application.properties. But if not, let me know this and i will change it)

Reply:
Yes, move them to application.staging.properties and remove from application.properties.example

Reply:
I changed it, check please